ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -142,23 +127,46 @@ export class Pay extends IronfishCommand {
       }
     }
 
-    if (!from) {
-      const response = await client.getDefaultAccount()
-      const defaultAccount = response.content.account
+    if (!isValidPublicAddress(to)) {
+      this.log(`A valid public address is required`)
+      this.exit(1)
+    }
 
-      if (!defaultAccount) {
-        this.error(
-          `No account is currently active.
-           Use ironfish accounts:create <name> to first create an account`,
-        )
+    if (fee == null) {
+      let suggestedFees: string = ""
+      try {
+        const response = await client.estimateFee({ 
+          fromAccountName: from, 
+          receives:  [{
+            publicAddress: to,
+            amount: CurrencyUtils.encode(amount),
+            memo: memo,
+          }]
+      })
+        suggestedFees.concat(`low: ${CurrencyUtils.renderIron(response.content.low)}\n`)
+        suggestedFees.concat(`medium: ${CurrencyUtils.renderIron(response.content.medium)}\n`)
+        suggestedFees.concat(`high: ${CurrencyUtils.renderIron(response.content.high)}\n`)
+      } catch {
+        suggestedFees = ""
       }
 
-      from = defaultAccount.name
+      const input = (await CliUx.ux.prompt(
+        `Enter the fee amount in $IRON (min: ${CurrencyUtils.renderIron(1n)} \n
+        recommended: ${suggestedFees}`,
+        {
+          required: true,
+        },
+      )) as string
+
+      if (!CurrencyUtils.isValidIron(input)) {
+        this.error(`A valid amount is required`)
+      }
+
+      fee = CurrencyUtils.decodeIron(input)
     }
 
-    if (!isValidPublicAddress(to)) {
-      this.log(`A valid public address is required`)
-      this.exit(1)
+    if (fee < 1n) {
+      this.error(`The minimum fee is ${CurrencyUtils.renderOre(1n, true)}`)
```

Comment:
I think the output of line 180:
```
${CurrencyUtils.renderIron(amount, true)} plus a transaction fee of ${CurrencyUtils.renderIron(
```
Was outputting an incorrect value when Jason and I were poking around, can you add test output to PR?

Reply:
This is what I see testing:
```
You are about to send:
$IRON 0.00010000 plus a transaction fee of $IRON 0.00000652 to . . .
```

Reply:
I am getting expected output
```
You are about to send:
$IRON 10.00000000 plus a transaction fee of $IRON 0.00000825 to acc03c941600e99cb0e53d9062826071a00af58905606aaabc4afc57d9626855b58724394b1a05d0d4a742 from the account default

* This action is NOT reversible *
```